packages/sandboxes/hello-world/.codesandbox/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM node:18-bullseye
```

Comment:
Our root `package.json` defines all monorepo packages as matching this glob:

```json
  "workspaces": [
    "packages/*"
  ]
```

The files you've added here won't match that, because they're two levels deep.

To resolve this, I think you can do `"packages/**"`, or `"packages/*", "packages/sandboxes/*"`.

Reply:
Then you'll want to apply this diff to make `yarn test` work:

```diff
diff --git a/packages/sandboxes/hello-world/package.json b/packages/sandboxes/hello-world/package.json
index c1759b0..0da9253 100644
--- a/packages/sandboxes/hello-world/package.json
+++ b/packages/sandboxes/hello-world/package.json
@@ -5,7 +5,8 @@
   "license": "MIT",
   "scripts": {
     "typecheck": "tsc",
-    "test": "npm run typecheck && tsx index.tsx"
+    "test": "npm run typecheck",
+    "start": "npm run test && tsx index.tsx"
   },
   "dependencies": {
     "ai-jsx": "^0.2.0-1",
@@ -18,4 +19,3 @@
     "typescript": "^5.1.3"
   }
 }
-
diff --git a/packages/sandboxes/hello-world/tsconfig.json b/packages/sandboxes/hello-world/tsconfig.json
index 4d02c7c..456f427 100644
--- a/packages/sandboxes/hello-world/tsconfig.json
+++ b/packages/sandboxes/hello-world/tsconfig.json
@@ -1,6 +1,6 @@
 {
   "extends": "@tsconfig/node18/tsconfig.json",
-  "include": ["src/*.tsx"],
+  "include": ["index.tsx"],
   "compilerOptions": {
     "jsx": "react",
     "jsxFactory": "LLMx.createElement",
@@ -11,4 +11,3 @@
     "noEmit": true
   }
 }
-
```